packages/ui/src/kit/VRadio/VRadio.vue

```diff
@@ -14,10 +14,16 @@
     >
       <slot />
     </span>
+    <tooltip-molecule
+      :label="tooltipLabel"
+      :disabled="true"
+      v-bind="$attrs"
```

Comment:
please, do no use `v-bind="$attrs"`
all you need here is just pass `label` prop and that's all

Reply:
here for now this is not required, but in general it's okay and common practice

Reply:
We have a lot of bugs in tests (when passing data-test) and when we pass another prop and don't want to have prop dublicates